config.md

```diff
@@ -1,13 +1,12 @@
 # <a name="containerConfigurationFile" />Container Configuration file
 
-The container's top-level directory MUST contain a configuration file called `config.json`.
```

Comment:
Did you mean to drop this line? Maybe you should include it below the moved paragraph?